scipy/linalg/_basic.py

```diff
@@ -298,7 +298,7 @@ def solve_triangular(a, b, trans=0, lower=False, unit_diagonal=False,
     Returns
     -------
     x : (M,) or (M, N) ndarray
-        Solution to the system `a x = b`.  Shape of return matches `b`.
+        Solution to the system ``a x = b``.  Shape of return matches `b`.
```

Comment:
and here

Reply:
for simple equation like that I believe that for no double backticks are better as IPython `??` will show the raw docstring. I think that `:math:` is warranted only when latex/formatting makes reading easier in HTML. I'm working on having :math: render in IPython, but it will take time. 

Though if you want me to change happy to do it, right now i'm focus on having single backtick with de default role of py:obj not failing.

Reply:
okay, no worries then!